application/src/main/java/org/thingsboard/server/config/ThingsboardSecurityConfiguration.java

```diff
@@ -82,6 +84,9 @@ public class ThingsboardSecurityConfiguration {
     public static final String MAIL_OAUTH2_PROCESSING_ENTRY_POINT = "/api/admin/mail/oauth2/code";
     public static final String DEVICE_CONNECTIVITY_CERTIFICATE_DOWNLOAD_ENTRY_POINT = "/api/device-connectivity/mqtts/certificate/download";
 
+    @Value("${server.http.max_payload_size:/api/image*/**=52428800;/api/**=16777216}")
```

Comment:
As we discovered, for multipart files the content length is -1, so we can exclude it from this list